packages/genesys-spark-tokens/data/tokens.json

```diff
@@ -4564,7 +4720,7 @@
     },
     "badge": {
       "border_radius": {
-        "value": "{badge.height} / 2",
+        "value": "{border_radius.full}",
```

Comment:
I would be surprised if this change is correct.
![badges](https://github.com/MyPureCloud/genesys-spark/assets/69190681/0fab5e24-a8f9-4ae5-a6e0-68b9792642b9)